contrib/completion/bash/docker

```diff
@@ -5554,11 +5571,15 @@ _docker() {
 		wait
 	)
 
+	local known_plugin_commands=(
```

Comment:
Should we somehow detect if it's installed (to prevent it from providing (or trying to provide) completion for the plugin when it's not installed, which I _guess_ would fail)

Reply:
I agree that this completion should only be added if the plugin is installed.